doc/rst/meta/modules/packages.rst

```diff
@@ -19,6 +19,7 @@ Algorithms
    :maxdepth: 1
 
    Crypto <packages/Crypto>
+   NPBRand <packages/NPBRand>
```

Comment:
IMO `NPBRandom` is a more obvious name for the module

Reply:
I think I ran into ambiguity conflicts with the existing submodule when I tried that name. Any idea how to get around that?

Reply:
I don't think the existing submodule needs to exist anymore, since you are moving this code (without changing it). Since there will only be one NPBRandom, it can be a toplevel module. In `module Random` it will still have `public use NPBRandom` and as a result it will still appear available as `Random.NPBRandom`.